go/vt/vttablet/tabletmanager/rpc_replication.go

```diff
@@ -706,14 +707,36 @@ func (tm *TabletManager) setReplicationSourceRepairReplication(ctx context.Conte
 		return err
 	}
 
+	currentPrimary, err := tm.TopoServer.GetTablet(ctx, parentAlias)
```

Comment:
isn't this a duplicate of the code block right above it?
```
parent, err := tm.TopoServer.GetTablet(ctx, parentAlias)
...
```

Reply:
@vmogilev the compiler won't allow that to happen. Here is the full block: https://github.com/slackhq/vitess/blob/46a352662e5aa086e1914c88c1d786126c4db413/go/vt/vttablet/tabletmanager/rpc_replication.go#L704-L740

Reply:
I misunderstood and see the problem now. Updated